rewrite-yaml/src/test/java/org/openrewrite/yaml/tree/MappingTest.java

```diff
@@ -400,34 +400,30 @@ void mappingAnchor() {
       " '\\n' ",
       " '\n' ",
       " \n ",
-      " \"\\.\" ",
```

Comment:
Note: I've removed the escape sequences which are illegal according to the SnakeYAML parser.